e2e-tests/portal/cypress/e2e/pages/utilisation-report-service/record-corrections/provideCorrection.js

```diff
@@ -0,0 +1,16 @@
+const page = {
+  facilityIdInput: () => cy.get('input[data-cy="FACILITY_ID_INCORRECT-input"]'),
```

Comment:
could these be lowercase to be consistent?

Reply:
The first part is using the record correction reason enum value so the component test assertions can be simpler (eg when we want to check that only the input for a single reason is showing we can get the difference to get a list of reasons that shouldn't show then iterate over those, passing in the enum value to the data-cy attribute).

Can change these to lowercase and refactor if needed, or maybe change to use the constant (below)? No strong preference

```suggestion
  facilityIdInput: () => cy.get(`input[data-cy="${RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT}-input"]`),
```

Thanks

Reply:
Just worried that if we change the ENUM of the reason, then this will be out of date.  maybe we can tackle this later with a change in the njk.  This is fine for now